src/PlusImageProcessing/Testing/vtkPlusTransverseProcessEnhancerTest.cxx

```diff
@@ -36,10 +62,130 @@ int main(int argc, char **argv)
 
   vtkPlusLogger::Instance()->SetLogLevel(verboseLevel);
 
-  ///////////////
+  //check for various non-optional command line arguments
+  int commandCheckStatus = NULL;
+
+  if (inputFileName.empty())
+  {
+    std::cerr << "--input-seq-file not found!" << std::endl;

```

Comment:
the message is not accurate, it suggests that some filename was specified but that file was not found; instead write something like --input-seq-file parameter is required. Same for all similar arguments.